apps/system/js/wrapper_factory.js

```diff
@@ -182,8 +191,10 @@
 
     generateBrowserConfig: function wf_generateBrowserConfig(features) {
       var config = {};
+      config.chrome = {};
       config.title = features.name;
       config.icon = features.icon || '';
+      config.chrome.pinned = features.pinned || false;
```

Comment:
That's not entirely true. This config is the initial configuration for the chrome on a new window. We'll need (somehow) to specifiy to a new created window that the initial state is 'pinned'. Not sure if the features object is the best way, but couldn't think in another one for now. That doesn't mean that the reConfig method in the AppChrome cannot reset the pinned config in that chrome depending on the url that is loaded afterwards.